Fundamentals/Design/Partial-View-Macro-Files/index.md

```diff
@@ -46,7 +46,28 @@ To create a Partial View Macro File from the snippet, go to the **Settings** sec
 
 ![Created partial view from snippet](images/created-partial-view-macro-file-from-snippet.png)
 
-By default, the partial view is saved in the `Views/MacroPartials` folder.
+By default, the partial view is saved in the `Views/MacroPartials` folder. Umbraco provides the following snippets:
```

Comment:
Would it make sense to say something like: "Umbraco provides the following Macro Partial snippets"?

What do you think? :)

Reply:
Makes sense 😀